plasmapy/physics/dimensionless.py

```diff
@@ -1,7 +1,25 @@
-""" 
+"""
 Module of dimensionless plasma parameters.
 
 These are especially important for determining what regime a plasma is
 in. (e.g., turbulent, quantum, collisional, etc.).
 
 """
+
+import numpy as np
+from astropy import units
+
+from ..constants import (k_B,
+                         )
+from .quantum import (Fermi_energy,
```

Comment:
Absolute imports would be quite neat here as well.

Reply:
done.